### PR TITLE
Orthographic projection

### LINKED ISSUE #311: Implement orthographic projection
Still to be done:
- [x] change label of the toggle action dynamically ("orthographic projection" / "perspective projection")
- [x] calculate suitable distance_to_target when loading a structure

### PR DESCRIPTION
**Pull request by adrianusler**
Closes #311. I need to fix the coverage; once this is done, it is ready to merge!

1. One can now toggle between perspective and orthographic projection
2. I have implemented that the zoom of the window (`distance_to_target`) is set automatically to match the dimensions of the loaded structure. Before this, the camera was often much too close to the structure (or within it) when it was loaded.

### CODE REVIEW

**Comment by Michel-Heinz** [comment]
I found a bug! When I load a trajectory move the camera around, and then load the next step, the camera resets. This does not happen in the main branch. It must has something to do with your changes in the camera module.

**Comment by adrianusler** [comment]
Thanks, I will take care of this.

File: src/molara/Gui/structure_widget.py [hunk 1]
```diff
@@ -140,7 +140,11 @@ def set_structure(self, struct: Structure | Crystal | Molecule, reset_view: bool
         """
         self.structure = struct
         self.structure_is_set = True
-        self.reset_view() if reset_view else self.center_structure()
+        if reset_view:
+            self.reset_view()
+        else:
+            self.set_vertex_attribute_objects()
```
Comment (adrianusler) [inline src/molara/Gui/structure_widget.py:146]: What is this for?

**Comment by Michel-Heinz** [comment]
I am fine with merging now!

**Review by Michel-Heinz — APPROVED** [review]
👍

**Pull request merged**